Utilities/Thread.h

```diff
@@ -18,15 +18,15 @@
 [[noreturn]] void catch_all_exceptions();
 
 // Hardware core layout
-enum class native_core_arrangement : u32
+enum class native_core_arrangement : u64
 {
 	undefined,
 	generic,
 	intel_ht,
 	amd_ccx
 };
 
-enum class thread_class : u32
+enum class thread_class : u64
```

Comment:
unnecessary

Reply:
ah yes, is my test i have forget remove it ^^